client/analytics/settings/historical-data/progress.js

```diff
@@ -3,20 +3,25 @@
  * External dependencies
  */
 import { __, sprintf } from '@wordpress/i18n';
+import { isNil } from 'lodash';
 
 function HistoricalDataProgress( { label, progress, total } ) {
+	let text = sprintf( __( 'Imported %(label)s', 'woocommerce-admin' ), {
+		label,
+	} );
+
+	if ( ! isNil( progress ) && ! isNil( total ) ) {
+		text +=
+			' ' +
```

Comment:
I know this wasn't introduced in this PR, but concatenation like this breaks proper RTL translations.

Reply:
Thanks for the review @jeffstieler! It should be fixed here:

a43186d6f832d686e65b65c6f336ad34a538bae1